service/sql/model.go

```diff
@@ -2416,16 +2438,24 @@ type StatementParameterListItem struct {
 // successful, and statement closed; result no longer available for fetch
 type StatementState string
 
+// user canceled; can come from explicit cancel call, or timeout with
```

Comment:
This is an example change where the enum descriptions are now correctly generated. Previously, //openapi library did not parse - or + bullets.